install_config/router/customized_haproxy_router.adoc

```diff
@@ -199,16 +627,43 @@ After this modification, you can xref:rebuilding-your-router[rebuild your router
 [[rebuilding-your-router]]
 == Rebuilding Your Router
 
-After you have made any desired modifications to the template, such as the
-example xref:using-stick-tables[stick tables] customization, you must rebuild
-your router for your changes to go in effect:
-
-. https://access.redhat.com/articles/881893#createimage[Rebuild the container
-image to include your customized template.]
-. xref:../registry/accessing_registry.adoc#access[Push the resulting image to your repository].
-. Create the router specifying your new image, either:
-.. in the pod's object definition directly, or
-.. by adding the `--images=<repo>/<image>:<tag>` flag to the `oadm router`
-command when
-xref:../../admin_guide/high_availability.adoc#configuring-a-highly-available-service[creating
-a highly-available routing service].
+In order to rebuild the router you need copies of several files that are present
+on a running router. Make a work directory and copy the files from the router:
+
+
+----
+# mkdir - myrouter/conf
+# cd myrouter
+# oc get po
+NAME                       READY     STATUS    RESTARTS   AGE
+router-2-40fc3             1/1       Running   0          11d
+# oc rsh router-2-40fc3 cat haproxy-config.template > conf/haproxy-config.template
+# oc rsh router-2-40fc3 cat error-page-503.http > conf/error-page-503.http
+# oc rsh router-2-40fc3 cat default_pub_keys.pem > conf/default_pub_keys.pem
+# oc rsh router-2-40fc3 cat ../Dockerfile > Dockerfile
+# oc rsh router-2-40fc3 cat ../reload-haproxy > reload-haproxy
+----
+
+You can edit or replace, any of these files. However, conf/haproxy-config.template
+and reload-haproxy are the most likely to be modified.
+
+After updating the files:
+
+----
+# docker build -t openshift/origin-haproxy-router-myversion .
+# docker tag openshift/origin-haproxy-router-myversion 172.30.243.98:5000/openshift/haproxy-router-myversion <1>
+# docker push 172.30.243.98:5000/openshift/origin-haproxy-router-pc:latest <2>
+----
+<1> Tag the version with the repository. In this case the repository is
+172.30.243.98:5000
+<2> Push the tagged version to the repository. It may be necessary to docker
+login to the repository first.
+
+To use the new router either edit the router deployment configuration by
+changing the *image:* string, or by adding the `--images=<repo>/<image>:<tag>`
+flag to the `oadm router` command.
```

Comment:
"To use the new router, edit the router deployment configuration either by changing the *image:* string or by adding" (add a comma after the introductory phrase and re-arrange either/or for parallelism).